packages/core/src/custom-values.ts

```diff
@@ -252,7 +257,7 @@ export function getBoxValue(
         return value;
     } else if (value && customTypes[value.type]) {
         return customTypes[value.type].getValue(path, value, node, customTypes);
-    } else if (value.type === 'string') {
+    } else if (value.type === 'st-string') {
         return (value as Box<'string', string>).value;
```

Comment:
shouldn't it create a `st-box` type or use the `boxString` function?

Reply:
It should unbox it, I changed it to use `unbox`